src/ArrayHelper.php

```diff
@@ -378,6 +378,143 @@ public static function setValue(array &$array, array|float|int|string|null $key,
         $array[self::normalizeArrayKey(array_shift($keys))] = $value;
     }
 
+    /**
+     * Appends a value to an associative array at the key path specified.
+     * If there is no such key path yet, it will be created recursively and an empty array will be initialized.
+     *  If the value exists, it will become the first element of the array
```

Comment:
```suggestion
     * If the value exists, it will become the first element of the array
```

Please check spaces for each comment line